app/gui/src/dashboard/components/AriaComponents/Dialog/Dialog.tsx

```diff
@@ -558,10 +558,7 @@ const DialogHeader = React.memo(function DialogHeader(props: DialogHeaderProps)
   }, [handleScrollEvent, scrollerRef])
 
   return (
-    <aria.Header
-      ref={headerDimensionsRef}
-      className={styles.header({ scrolledToTop: isScrolledToTop })}
-    >
+    <header ref={headerDimensionsRef} className={styles.header({ scrolledToTop: isScrolledToTop })}>
```

Comment:
it's probably *fine* but why the reason for the change?

Reply:
Well there was a collision when we try to wrap a menu item with a Dialog.trigger and put dialog as a sibling.

But because the menu closes on click, dialog is also dissapears. so it's no longer a case

Reply:
Reverted